src/breadcrumbs/breadcrumbs.scss

```diff
@@ -57,7 +57,7 @@
 @mixin iui-breadcrumbs-separator {
   display: flex;
 
-  svg {
+  .iui-icon {
```

Comment:
Revert

Reply:
Reverted changes in breadcrumbs SCSS and HTML in next push.